README.md

```diff
@@ -33,15 +33,26 @@ These configuration keys are currently mandatory:
 
 | Variable  | Example  | Description                                                                              |
 |-----------|----------|------------------------------------------------------------------------------------------|
-| VIP_IP    | 10.1.2.3 | Virtual IP that is being managed                                                         |
-| VIP_MASK  | 24       | Netmask of the virtual IP                                                                |
-| VIP_IFACE | eth0     | Network interface to configure the IP address on. Usually the primary network interface. |
-| VIP_KEY   | /service/batman/leader | Key to monitor. Must match  scope from Patroni postgres.yml                |
-| VIP_HOST  | serverX  | Value to trigger on. Must match name from Patroni.                                       |
-| VIP_TYPE  | etcd     | Type of endpoint (etcd or consul)                                                        |
-| VIP_ENDPOINT | http://10.1.2.3:2379 | Location of endpoint (etcd or consul)                                     |
-
-
+| IP    | 10.1.2.3 | Virtual IP that is being managed                                                         |
+| MASK  | 24       | Netmask of the virtual IP                                                                |
+| IFACE | eth0     | Network interface to configure the IP address on. Usually the primary network interface. |
+| KEY   | /service/batman/leader | Key to monitor. Must match `<namespace>/<scope>/leader from Patroni.                |
+| HOST  | serverX  | Value to trigger on. Must match `<name>` from Patroni.                                       |
+| TYPE  | etcd     | Type of endpoint (etcd or consul)                                                        |
+| ENDPOINT | http://10.1.2.3:2379 | Location of endpoint (etcd or consul)                                     |
```

Comment:
the `VIP_` prefix was there because the first versions of vip-manager used Environment variables, the prefix was there to avoid conflicts with other ENV vars, as IP or KEY etc. are very generic...

Maybe we should update this table to include both the ENV option name and the yaml option name.